virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -3,70 +3,53 @@
 ====
 
-include::modules/odc-creating-vms.adoc[leveloffset=+1]
+Virtual machine templates with a boot source are labeled *Available boot source* or they display a customized label text. Using templates with an available boot source expedites the process of creating virtual machines.
 
-include::modules/virt-creating-vm-quick-start-web.adoc[leveloffset=+1]
+Templates without a boot source are labeled *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
 
-include::modules/virt-quick-creating-vm.adoc[leveloffset=+1]
+include::modules/virt-creating-vm-template.adoc[leveloffset=+2]
 
-include::modules/virt-creating-vm-custom-template.adoc[leveloffset=+1]
+[NOTE]
+====
+See xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-vm-storage-volume-types_virt-create-vms[storage volume types] and xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-storage-wizard-fields-web_virt-create-vms[storage fields] for details about disk sources.
+====
 
-Refer to the virtual machine fields section when creating a VM from the web console.
+include::modules/virt-vm-storage-volume-types.adoc[leveloffset=+3]
 
 :virtualmachine:
-include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
-
-include::modules/virt-storage-wizard-fields-web.adoc[leveloffset=+2]
+include::modules/virt-storage-wizard-fields-web.adoc[leveloffset=+3]
+:!virtualmachine:
 
-include::modules/virt-cloud-init-fields-web.adoc[leveloffset=+2]
-
-To configure storage class defaults, use storage profiles. For more information, see xref:../../virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc#virt-customizing-storage-profile_virt-creating-data-volumes[Customizing the storage profile].
-
-include::modules/virt-creating-vm-yaml-web.adoc[leveloffset=+2]
+include::modules/virt-creating-vm-instancetype.adoc[leveloffset=+2]
 
 include::modules/virt-creating-vm-cli.adoc[leveloffset=+1]
 
-:virtualmachine!:
-
-include::modules/virt-vm-storage-volume-types.adoc[leveloffset=+1]
-
-include::modules/virt-about-runstrategies-vms.adoc[leveloffset=+1]
-
-include::modules/virt-creating-vm-instancetype.adoc[leveloffset=+1]
-
+// This should probably be moved somewhere else because it's a config.
+include::modules/virt-about-runstrategies-vms.adoc[leveloffset=+2]
 
 [id="additional-resources_virt-create-vms_{context}"]
 [role="_additional-resources"]
 == Additional resources
 
 * The `VirtualMachineSpec` definition in the link:https://kubevirt.io/api-reference/{KubeVirtVersion}/definitions.html#_v1_virtualmachinespec[KubeVirt {KubeVirtVersion} API Reference] provides broader context for the parameters and hierarchy of the virtual machine specification.
```

Comment:
Because this bullet point is not "additional resources" compliant, but it's still useful, maybe we could move it somewhere else?

Reply:
Sorry, I don't understand. What is "additional resources" compliant?

Reply:
Thanks. I got that from the link you sent to the OSG section, so I fixed the additional resources.